src/app/Fake.IO.FileSystem/Directory.fs

```diff
@@ -0,0 +1,49 @@
+namespace Fake.IO.FileSystem
```

Comment:
Maybe these modules should move to `Fake.IO` :/

Reply:
no preferences...

Reply:
Yes I think we should move them (I can do that if you want), because then it is more similar to `System.IO` and one less `open Fake.IO.FileSystem` at the top...